docker-compose.yml

```diff
@@ -21,7 +22,8 @@ services:
     - OPENVPN_PASSWORD=password
     - OPENVPN_OPTS="--inactive 3600 --ping 10 --ping-exit 60"
     - LOCAL_NETWORK=192.168.0.0/24
-
+    - WEBPROXY_ENABLED=true
```

Comment:
Since this is a opt-in feature, I think we should remove it from the default docker-compose and add description in README that the port needs to be opened as well.

Reply:
Thanks for looking at this. I am keen to dispense with my custom branch, I'll have a crack at this and test this weekend.

Reply:
Great! One more thing I got to think of. You asked if there was a way to make the container go down when the vpn dropped. It should do that if you set the `OPENVPN_OPTS="--inactive 3600 --ping 10 --ping-exit 60"` option. That should exit the openvpn process when it's not able to ping anymore, and setting `--restart=always` or `--restart=unless-stopped` should make the container restart itself.

I've seen that config for some providers have the `ping-restart` option set. That should probably be removed to ensure that it actually shuts down.